scripts/download_jsut.py

```diff
@@ -0,0 +1,11 @@
+import os
+import pathlib
+import subprocess
+
+url = "http://ss-takashi.sakura.ne.jp/corpus/jsut_ver1.1.zip"
+output_dir = pathlib.Path(__file__).parents[1].joinpath("data")
+if not os.path.exists(output_dir):
+    os.makedirs(output_dir)
```

Comment:
変数 `output_dir` は `<class 'pathlib.PosixPath'>`なので、pathlibのメソッドを使って置き換えることができます。
```suggestion
if not output_dir.exists():
    output_dir.mkdir(parents=True)
```